torch_xla/distributed/fsdp/_init_utils.py

```diff
@@ -0,0 +1,102 @@
+import collections
+import warnings
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Generator,
+    Iterable,
+    Iterator,
+    List,
+    no_type_check,
+    Optional,
+    Set,
+    Tuple,
+    Type,
+    Union,
+)
+
+import torch
+import torch.nn as nn
+import torch_xla.core.xla_model as xm
+
+from .xla_flatten_params_wrapper import FlatParameter
+
+_TORCHDISTX_AVAIL = True
+try:
+  from torchdistx import deferred_init, fake  # type: ignore[import]
+except ImportError:
+  _TORCHDISTX_AVAIL = False
+
+
+def _materialize_module(
+    module: nn.Module,
+    param_init_fn: Optional[Callable[[nn.Module], None]],
+    ignored_params: Set[nn.Parameter],
```

Comment:
yes I leave here in case we need it in the future. The auto_wrap also has ignored_params interface but not sued, so theoretically we can leverage them together

Reply:
Maybe add a TODO?